skops/hf_hub/_hf_hub.py

```diff
@@ -94,6 +169,15 @@ def push(*, repo_id: str, source: Union[str, Path], token: str = None):
         A token to push to the hub. If not provided, the user should be already
         logged in using ``huggingface-cli login``.
 
+    commit_message: str, optional
+        The commit message to be used when pushing to the repo.
+
+    create_remote: bool, optional
+        Whether to create the remote repository if it doesn't exist. If the
+        remote repository doesn't exist and this parameter is ``False``, it
+        raises an error. Otherwise it checks if the remote repository exists,
+        and would create it if it doesn't.
```

Comment:
If this exists, why do we create_repo in examples and tests?

Reply:
I've removed it from the example, but the test needs to test different scenarios, and the user might create the repo before calling this method, so the test makes sure that case is tested as well.